src/main/java/pl/targosz/homework/InterfaceToCountParimeterAndArea/Square.java

```diff
@@ -0,0 +1,18 @@
+package pl.targosz.homework.InterfaceToCountParimeterAndArea;
+
+public class Square implements InterfaceToCountParimeterAndArea{
+    double a;
```

Comment:
To pole powinno być private (analogicznie w przypadku innych klas w tym commicie). Ustawić powinieneś je za pomocą konstruktora. ;)

https://1024kb.pl/nauka-java/hermetyzacja-java/